src/control/mod.rs

```diff
@@ -857,13 +858,15 @@ pub trait Device: super::Device {
         Ok(from_u32(handle).unwrap())
     }
 
-    /// Imports a sync file exported by [`Self::syncobj_to_sync_file`] into an existing process-local handle.
+    /// Imports a sync file exported by [`Self::syncobj_to_sync_file`] into an existing process-local handle
+    /// optionally for a specified timeline point.
     fn sync_file_into_syncobj(
         &self,
         fd: BorrowedFd<'_>,
         handle: syncobj::Handle,
+        timeline_point: Option<u64>,
```

Comment:
If I'm not mistaken, this timeline point is "on" the sync file `fd` (rather than on the target syncobj `handle`), so it would make more sense for this argument to be next to `fd`

Reply:
It's on the handle. The `syncobj` can be a timeline and `sync_file` would then be the `dma_fence` that is waited on or signaled at the given timeline point.


https://docs.kernel.org/gpu/drm-mm.html#drm-sync-objects

Reply:
? just making sure, it's on the `handle` so the current position makes sense right? It's not on the `fd`, which "simply" a binary `sync_file`.